CONTRIBUTING.rst

```diff
@@ -21,8 +21,8 @@ In order to add a feature:
 - The feature must be documented in both the API and narrative
   documentation.
 
-- The feature must work fully on the following CPython versions:  2.7,
-  3.5, 3.6, 3.7 and 3.8 on both UNIX and Windows.
+- The feature must work fully on the following CPython versions:
```

Comment:
I think this library still supports 2.7 so the CONTRIBUTING.rst may need to be excluded temporarily.

Reply:
Done.